cal.sh

```diff
@@ -46,4 +49,7 @@ elif [ $selection == "3" ]; then
     mul $firstNumber $secondNumber
 elif [ $selection == "4" ]; then
     div $firstNumber $secondNumber
-fi
\ No newline at end of file
+elif [ $selection == "5" ]; then
+    reminder $firstNumber $secondNumber
+
```

Comment:
@Arunmeethu  please clean up this whitespace

Reply:
Removed empty spaces.